pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/ControllerRequestBuilderUtil.java

```diff
@@ -59,8 +59,8 @@ public static void addFakeBrokerInstancesToAutoJoinHelixCluster(String helixClus
       helixZkManager.connect();
       if (isSingleTenant) {
         helixZkManager.getClusterManagmentTool()
-            .addInstanceTag(helixClusterName, brokerId, TenantNameBuilder.getBrokerTenantNameForTenant(
-                TenantNameBuilder.DEFAULT_TENANT_NAME));
+            .addInstanceTag(helixClusterName, brokerId,
+                TagNameBuilder.getBrokerTagForTenant(TagNameBuilder.DEFAULT_TENANT_NAME));
```

Comment:
why did we move the constant from tenantNameBuilder to tagNameBuilder.

Reply:
The comment in the commit indicates why. It is not really a Tenant name builder, it is a Tag name builder. So the class was changed. @npawar forgot to delete the old class, I will delete it in the next commit to this branch.